My Simple App/Application/WeatherTableViewCell.swift

```diff
@@ -9,20 +9,104 @@
 import UIKit
 
 class WeatherTableViewCell: UITableViewCell {
+  var titleLabel: UILabel = {
+    let label = UILabel()
+    // When you're writing programatic UI using anchors, this property has to
+    //   be disabled on all views. At larger dev shops, there will likely be some sort
+    //   of view framework and system views (eg UILabel, UIButton) will not be used
+    //   directly. The wrapper views will disable this property for you in that case.
+    label.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
since this (and some of the other views) are added to a stack view, `translatesAutoresizingMaskIntoConstraints ` is set to false for you:

```
let a = UIView()
let b = UIView()
let sv = UIStackView()
sv.addArrangedSubview(a)

print(a.translatesAutoresizingMaskIntoConstraints) // false
print(b.translatesAutoresizingMaskIntoConstraints) // true
```